GoogleForCreators/web-stories-wp
# Autosaves: Delete local backup after auto-saving to DB

## Context



## Summary
This addresses an issue with local auto-saving where local backup is not removed after auto-saving to database happens.


## Relevant Technical Choices



## To-do

- [x] Remove local backup when a story is auto-saved



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Add a new Story and publish.
2. Add some element
3. Wait for the local backup to be stored.
4. Now wait until the DB auto-saving happens
5. Verify that the local backup is removed once it happens.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Partially addresses #12547
